src/components/metagoofil/metagoofil.tsx

```diff
@@ -85,15 +88,21 @@ export function Metagoofil() {
         setLoading(true);
 
         const args = [`-d`, `${values.webname}`, `-t`, `${values.filetype}`];
-
+        //number of searches made
```

Comment:
if statement is fine but maybe hard to scale later down the line. Maybe consider a new logic that can scale for the future.